fastlane_bot/events/exchanges/solidly_v2.py

```diff
@@ -11,10 +11,18 @@
 from web3.contract import Contract
 
 from fastlane_bot.data.abi import SOLIDLY_V2_POOL_ABI, VELOCIMETER_V2_FACTORY_ABI, SOLIDLY_V2_FACTORY_ABI, \
-    VELOCIMETER_V2_POOL_ABI
+    VELOCIMETER_V2_POOL_ABI, SCALE_V2_FACTORY_ABI, EQUALIZER_V2_POOL_ABI
 from fastlane_bot.events.exchanges.base import Exchange
 from fastlane_bot.events.pools.base import Pool
 
+FEE_RESOLUTION_MAP = {
+    "velocimeter_v2": 10000,
+    "equalizer_v2": 10000,
+    "aerodrome_v2": 10000,
+    "velodrome_v2": 10000,
+    "scale_v2": 1000000000000000000,
+    "solidly_v2": 10000,
+}
```

Comment:
Please use `1e18`, `1e5`, etc, so it is easier to spot typing mistakes.
Perhaps even better - you can use `18`, `5`, etc, and in the code itself - `10 ** FEE_RESOLUTION_MAP[x]`.

Reply:
Updated